modules/auth/views/login.php

```diff
@@ -17,39 +17,87 @@
 		}
 	?>
 
-	<hr>
+	<br />
```

Comment:
Can we remove all `<br />` tags from this function, and instead make spaces/margin/padding with css styling?